impl/src/main/java/org/ehcache/config/builders/CacheConfigurationBuilder.java

```diff
@@ -62,9 +65,9 @@
  * instance without modifying the one on which the method was called.
  * This enables the sharing of builder instances without any risk of seeing them modified by code elsewhere.
  */
-public class CacheConfigurationBuilder<K, V> implements Builder<CacheConfiguration<K, V>> {
+public class CacheConfigurationBuilder<K, V> implements FluentCacheConfigurationBuilder<K, V, CacheConfigurationBuilder<K, V>> {
```

Comment:
`org.ehcache.config.builders.CacheConfigurationBuilder#withEvictionAdvisor` should have `@Override` annotation.

Reply:
Indeed.